### DISCUSSION TITLE
Unable to login into Workstation instance on Putty

### DISCUSSION THREAD

**Discussion by ebhaskarreddy** [learndevopsonline/learndevopsonline#1574] [Q&A]
For workstation instance i have taken t3.small , while login into putty it shows timed out  , same time i tried with t3.micro it succeed , could you help me to resolve it

**Comment by sreedharm07** [comment]
Hi @ebhaskarreddy 
its not about t3 small and t3 micro. its about security group may be.
may be you forgot to attach security group when you created t3.small thats why its timed out error.

**Comment by ebhaskarreddy** [comment]
I kept it has allow-all for security group related

**Reply by sreedharm07** [reply]
can you try once again and if you get same error please ping your ip

**Comment by ebhaskarreddy** [comment]
Resolved it .thanks
